client/src/templates/Challenges/redux/execute-challenge-saga.js

```diff
@@ -128,6 +129,7 @@ function* executeChallengeSaga({ payload }) {
     const challengeComplete = testResults.every(test => test.pass && !test.err);
     if (challengeComplete) {
       playTone('tests-completed');
+      fireConfetti();
```

Comment:
I believe this'll run on every completed challenge. We only want it to run when a full block has been completed.

Reply:
done